onnxruntime/contrib_ops/cpu/nchwc_ops.cc

```diff
@@ -67,27 +67,41 @@ ONNX_CPU_OPERATOR_TYPED_NCHWC_KERNEL(
         .TypeConstraint("T", DataTypeImpl::GetTensorType<float>()),
     NchwcAveragePool);
 
-template <typename T>
-Status ReorderInput<T>::Compute(OpKernelContext* context) const {
+Status ReorderInput::Compute(OpKernelContext* context) const {
   const auto* X = context->Input<Tensor>(0);
```

Comment:
Any chance this will need to support double or float16 in the future?

Reply:
I haven't heard any demand for that, but if there were, more work would need to be done to the other ops anyway to support the extra types, so this is easily restored then. I got rid of the int8 types as those won't use NCHWc as the blocking format.